app/Http/Controllers/ProfileController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Validator;
+
+/**
+ * @OA\Info(title="Account Setting API", version="1.0.0")
+ */
+class ProfileController extends Controller
+{
+    public function __construct()
+    {
+        //Auth middleware
+    }
+
+    /**
+     * @OA\Post(path="/changepassword",
+     *   tags={"changepassword"},
+     *   summary="Update Password",
+     *   description="This is used to update the user password.",
+     *   operationId="changePassword",
+     *   @OA\Parameter(
+     *     name="current_password",
+     *     required=true,
+     *     in="query",
+     *     description="The current password of logged in user",
+     *     @OA\Schema(
+     *         type="string"
+     *     )
+     *   ),
+     *    @OA\Parameter(
+     *     name="new_password",
+     *     required=true,
+     *     in="query",
+     *     description="The new password of logged in user",
+     *     @OA\Schema(
+     *         type="string"
+     *     )
+     *   ),
+     *    @OA\Parameter(
+     *     name="confirm_password",
+     *     required=true,
+     *     in="query",
+     *     description="The confirm password same as new password of logged in user",
+     *     @OA\Schema(
+     *         type="string"
+     *     )
+     *   ),
+    *   @OA\Response(status_code=200, message="Password updated successfully"),
+    *   @OA\Response(
+    *         response=400,
+    *         message="Error",
+    *         @OA\JsonContent(
+    *             oneOf={
+    *                 @OA\Schema(ref="#/components/schemas/Error"),
+    *                 @OA\Schema(type="string")
+    *             },  
+    *         )
+    *     )
+    * )
+    */
+    public function changePassword(Request $request)
+    {
+        $user = User::where('email', '=', 'reenanalwa@gmail.com')->first();
+        $message = [
```

Comment:
A new class has been created ValidationMessages.php. Please move the messages into that class and use it here as `$this->messages->getChangePasswordMessages()` if the function is not there please create. In this way, we move the validation messages out of the controller.